projects/storefrontlib/src/cms-components/checkout/services/express-checkout.service.ts

```diff
@@ -0,0 +1,118 @@
+import { Injectable } from '@angular/core';
+import { combineLatest, of } from 'rxjs';
+import { filter, map, switchMap, withLatestFrom } from 'rxjs/operators';
+
+import {
+  Address,
+  CheckoutDeliveryService,
+  UserAddressService,
+  UserPaymentService,
+  PaymentDetails,
+  DeliveryMode,
+  CheckoutPaymentService,
+} from '@spartacus/core';
+import { CheckoutConfigService } from './checkout-config.service';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class ExpressCheckoutService {
+  shippingAddress$;
```

Comment:
Maybe rename to `shippingAddressSet$`

Also for `deliveryMode$` and `paymentMethod$`